### ISSUE TITLE
Can't access the full Documentation

### ISSUE THREAD

**Issue by filipegomes** [composes-toolkit/dissect#20]
The home page of the project do not load. How can I have access to the documentation? 

Both websites (project and toolkit) are inaccessible.

**Comment by L3viathan** [comment]
Hi, it appears to have been taken down, but Marco's personal website still has a mirror: http://marcobaroni.org/composes/toolkit

**Comment by filipegomes** [comment]
Hello, thanks for reply. I actually found his website but all links to documentation and tutorials are broken. All going to 404 not found.

**Comment by L3viathan** [comment]
Indeed. I found it in the Internet Archive though: https://web.archive.org/web/20181212114358/http://clic.cimec.unitn.it/composes/toolkit/introduction.html
